apps/onboarding-functions/src/main/java/it/pagopa/selfcare/onboarding/service/CompletionServiceDefault.java

```diff
@@ -217,6 +218,15 @@ public void persistActivatedAt(Onboarding onboarding) {
                 .where("_id", onboarding.getId());
     }
 
+    @Override
+    public void rejectOutdatedOnboardings(Onboarding onboarding) {
+        LocalDateTime now = LocalDateTime.now();
+        onboardingRepository
+                .update("status = ?1 and updatedAt = ?2 ", REJECTED, now)
+                .where("productId = ?1 and institution.origin = ?2 and institution.originId = ?3 and _id != ?4 (status = PENDING or status = TOBEVALIDATED)",
```

Comment:
Are we sure that an onaboarding pending have an institution.id?

Reply:
Yes, coz this function is invoked after the CREATE INSTITUTION activity. Therefore, it's not possible to use the only condition on status=PENDING coz the update to COMPLETED happens only after the entire block `executePendingState`